README.md

```diff
@@ -7,10 +7,65 @@ This workflow does the primary data processing for RNAseq data: alignment, QC, r
+
+
+| Header     | Description                                                                                                                                                                                                                                                                                                                                                                                                          |
+|------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| #chrom     | Number of the chromosome                                                                                                                                                                                                                                                                                                                                                                                             |
+| chromStart | Start position of the gene                                                                                                                                                                                                                                                                                                                                                                                           |
+| chromEnd   | End position of the gene                                                                                                                                                                                                                                                                                                                                                                                             |
+| gene_id    | Ensembl gene ID. For the exon-based featureCounts file this column states all Ensembl gene IDs for which the respective exon is part of (separated by “+”).                                                                                                                                                                                                                                                          
+|
+| score      | -                                                                                                                                                                                                                                                                                                                                                                                                                    |
```

Comment:
from here on the formatting does not fit anymore, if one uses the formatted representation of the README.md file

Reply:
Thanks, for spotting this. I'll try to check the Github rendering in the future, not just the one from the IDE.